app/models/shipit/deploy_spec/kubernetes_discovery.rb

```diff
@@ -16,8 +16,8 @@ def discover_task_definitions
               'action' => "Restart application",
               'description' => "Simulates a rollout of Kubernetes deployments by using kubernetes-restart utility",
               'steps' => [kubernetes_restart_cmd],
-            },
-          }
+            }
+          }.merge!(super)
```

Comment:
I'd like a test that make sure that explicitly defined tasks inside the `shipit.yml` have precedence.